upup/models/cloudup/resources/addons/metrics-server.addons.k8s.io/k8s-1.11.yaml.template

```diff
@@ -133,6 +130,8 @@ spec:
       containers:
       - args:
           - --secure-port=4443
+          - --kubelet-preferred-address-types=InternalIP,ExternalIP,Hostname
```

Comment:
Did you test this one? I am not sure if either InternalIP or ExternalIP is part of the kublet cert. Iirc we only add hostname.

Reply:
you are right ^^ these can only work when going insecure.

should we put them into insecure block or remove them completely?

Reply:
Maybe just explicitly set it to `Hostname` and add a comment why.

Reply:
i would suggest removing it completely and just let the `metrics-server` use its default behavior which is always Hostname --> DNS --> IPs as shown in [here](https://github.com/kubernetes-sigs/metrics-server/blob/master/pkg/utils/address_resolver.go#L24-L38) and avoid confusing maintainers. wdyt?

Reply:
also is this way, in edge cases where DNS is down, metrics-server would still operate fallback into IPs

Reply:
As long as the defaults do not change, that is fine :) with secure: true, the fallbacks won't work anyway though.